contrib/dhcp_agent/setup.py

```diff
@@ -0,0 +1,19 @@
+# -*- encoding: utf-8 -*-
+from setuptools import setup
+
+setup(
+    name='dhcp_agent',
+    version='0.0.1',
+    author='Grupa Allegro Sp. z o.o. and Contributors',
```

Comment:
ok, I will change also in https://github.com/allegro/ralph/blob/ng/setup.py#L16 :)